### ISSUE TITLE
Feature Request: Hide based on tags

### ISSUE THREAD

**Issue by ChrisJefferson** [Oliver-Akins/file-hider#18]
I would love to be able to hide all pages with `#archive` from the file explorer. I personally end up with files I don't want to delete, but don't want to see all the time.

**Comment by Oliver-Akins** [comment]
Good idea, I'm not sure how feasible this is with how the plugin works at the moment, but I'm looking into making the plugin a lot more robust, which would allow me to hopefully add this way easier than it would be now.
